crafters/image/ImageReader/README.md

```diff
@@ -1,3 +1,49 @@
 # ImageReader
 
-loads the image from the given file path and save the `ndarray` of the image in the Document. 
\ No newline at end of file
+**ImageReader** is a crafter that loads an image from a given file path and save the `ndarray` of the image in the Document. It reads the image specified in `buffer` and save the `ndarray` of the image in the `blob` of the document.
+
+The **ImageReader** executor needs only one parameter:
+
+| `param_name`  | `param_remarks` |
+| ------------- | ------------- |
+| `channel_axis`  |It's the desired audio signal to be normalized. |
```

Comment:
Is not an audi signal, is the color channel axis of the resulting image matrix

Reply:
Yes! it was mixed up with the audio file